minidump-processor/tests/test_processor.rs

```diff
@@ -82,7 +82,7 @@ fn test_processor() {
         assert_eq!(raw.eip, 0x0040429e);
         assert_eq!(*valid, MinidumpContextValidity::All);
     } else {
-        assert!(false, "Wrong context type");
+        panic!("Wrong context type");
```

Comment:
I'm not morally opposed to this but `panic!` just reads oddly in the context of tests. (I realize that `assert!` uses `panic!` internally.)